services/shhext/chat/rpc.go

```diff
@@ -28,3 +29,8 @@ type SendGroupMessageRPC struct {
 	Payload hexutil.Bytes
 	PubKeys []hexutil.Bytes
 }
+
+// JoinPublicChatRPC represents the RPC payload for the JoinPublicChat RPC method
+type JoinPublicChatRPC struct {
```

Comment:
`JoinPublicChatRPC` but contains plural `Chats`. Should we change it to `JoinPublicChatsRPC`?